pkg/util/mountOpts.go

```diff
@@ -49,9 +49,10 @@ func ProcessOptions(options []string, isTmpfs bool, sourcePath string) ([]string
 		case "O":
 			foundOverlay = true
 		case "idmap":
-			if len(options) > 1 {
```

Comment:
this check was added because overlay didn't support idmap and we would get a weird error message.

But given support for overlay idmapped mounts is being added to the upstream kernel, it is good to drop this check so we will be able to use overlay and idmapped mounts.